docs/parameters.md

```diff
@@ -1,246 +1,245 @@
-| **JVM_OPT_G1R_SET_UPDATING_PAUSE_TIME_PERCENT**          | Have the JVM do less remembered set work during STW, instead preferring concurrent GC. Reduces p99.9 latency.                                                                                                                                                                                                                             |                                                                                                                                                                                                   |
-| **CUSTOM_JVM_OPTIONS_BASE64**                            | Base64-encoded JVM options appended to jvm.options.                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                   |
-| **POD_MANAGEMENT_POLICY**                                | podManagementPolicy of the Cassandra Statefulset                                                                                                                                                                                                                                                                                          | OrderedReady                                                                                                                                                                                      |
-| **REPAIR_POD**                                           | Name of the pod on which 'nodetool repair' should be run.                                                                                                                                                                                                                                                                                 |                                                                                                                                                                                                   |
+| [1m                          Name                          [0m |                                                                                                                                                                Description                                                                                                                                                                |                                                                                              Default                                                                                              |
```

Comment:
Looks like some additional formatting was added to the table by an updated `pytablewriter`. Not sure if this is what we want. My environment is using version 0.54.0.

Reply:
I recall @zmalik said something about changing the version back a little, not sure it happened.

Reply:
The CI uses 0.51.0, I downgraded locally.